sa_sbl/CustomVariables.ini

```diff
@@ -1,8 +1,6 @@
 ;CustomVariables.ini
-;2 = player1
-2 = player_char
-;3 = scplayer
-3 = player_actor
```

Comment:
$PLAYER_ACTOR is no longer a thing?
What `scplayer` even stands for?

Reply:
It isn't the easiest decision to me, but it should help in the long run. This is how what they are called in the original scripts, and if we will be using them for reference, there will be less confusion.

Problem with old $PLAYER_CHAR is that, despite the name, it is not an instance of Char class. So if people will start using new classes/names more, it will be a source of constant confusion.

```
00A0: get_char_coordinates [Char] store_to x [float] y [float] z [float]
```
should I use PLAYER_CHAR here? 

PLAYER_ACTOR has its own problem with "ACTOR" not being used anywhere in SBL.

I know we have a 15+ years legacy behind, but with fresh modes it should be appropriate to use new variable names too.


In CLEO scripts, I would encourage using `0` instead of $player1 and `get_player_char` instead of $scplayer.

Reply:
` Player1` seems to be better. Maybe even "PlayerId_1'? 'Player.One', PlayerId.One', `Player.1'?
I never saw scplayer used anywhere, so how "original scripts" are even relevant. Still not explained what `scplayer` is short form of.

Reply:
from GTA III source code:
```
GET_PLAYER_CHAR player script_controlled_player
```

they shortened it to `scplayer` in other games

Reply:
Still weird. I would not glorify original sources so much, after all Rockstar scripts are just small percent of all scripts ever written for GTA :P
It sound logical that `[Char]` should take `$PLAYER_CHAR`, but that name was previously used for player id so it would lead to massive confusion. I do not think changing actor to char was good decision at all. Anyway, maybe then player's actor should be stored in variable `$PLAYER1_CHAR`

Not using player id variable at all makes sense, as player 1 ALWAYS has value `0`. I'm all for using `Player.One` constant instead.